<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThCli</class>
 <widget class="QDialog" name="ThCli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThCli</string>
  </property>
  <widget class="QLCDNumber" name="m_lcd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="numDigits">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_srvAddr">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_connect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
